src/boxes/movie.rs

```diff
@@ -124,17 +119,21 @@ impl Mp4Box for Movie {
             fields.push(mvhd.as_ref());
         }
 
-        for track in self.tracks.iter() {
-            fields.push(track.as_ref());
-        }
-
         if let Some(udta) = self.udta.as_ref() {
             fields.push(udta.as_ref());
         }
 
+        for track in self.tracks.iter() {
+            fields.push(track.as_ref());
+        }
+
         Some(fields)
     }
 
+    fn getmetadata(&self) -> Option<Vec<u64>> {
+        None
```

Comment:
If this is None everywhere except one place, it shouldn't be in the trait...

Reply:
This is the only pending issue, which I'm working on based on our in-person discussion. I will post a commit after this, and let you review the changed code.